yb-voyager/src/migassessment/sizing.go

```diff
@@ -214,21 +214,35 @@ func SizingAssessment(targetDbVersion *ybversion.YBVersion, sourceDBType string)
 		return fmt.Errorf("error fetching the sharded throughput: %w", err)
 	}
 
-	sizingRecommendationPerCore := createSizingRecommendationStructure(colocatedLimits)
+	sizingRecommendationPerCoreColocatedAndShardedCombined := createRecommendationStructureColocatedAndShardedCombined(colocatedLimits)
```

Comment:
could we group all the strategy operations together? It would be easier to follow, that way! 

i.e. first we have all the steps for the `colocatedAndShardedCombined` strategy up until we calculate the best recommendation for this strategy. And then, we have all the steps for the `allSharded` strategy. 
Finally, we choose the best between the two.